### ISSUE TITLE
Undefined index in class.jetpack-sync-functions.php 

### ISSUE THREAD

**Issue by richardmtl** [Automattic/jetpack#8137]
First reported in a forum thread: https://wordpress.org/support/topic/notice-undefined-index-48/

> Only in the administration area – Intermittent error:
> Notice: Undefined index: scheme in /xxxxxx/plugins/jetpack/sync/class.jetpack-sync-functions.php on line 210

> apache.error.log :
>PHP Notice: Undefined index: scheme in /xxx/plugins/jetpack/sync/class.jetpack-sync-functions.php on line 210\n', referer: https://xxxx/wp-admin/post.php?post=xxxxx&action=edit
> PHP Notice: Undefined index: scheme in /xxx/plugins/jetpack/sync/class.jetpack-sync-functions.php on line 210\n', referer: https://xxx/wp-admin/post-new.php

**Comment by Umangvaghela** [comment]
Hello @richardmtl  

I send the PR to resolve the issue so please review it when you get a time and provide your feedback for the same.
Thanks

**Comment by StefMattana** [comment]
Reopening as the same user reported a similar apache error log [here](https://wordpress.org/support/topic/trying-to-get-property-of-non-object-49/): 

`PHP message: PHP Notice: Trying to get property of non-object in xxxxxx/wp-content/plugins/jetpack/sync/class.jetpack-sync-module-posts.php on line 204\n’, referer: xxxx/…/wp-admin/post.php?post=xxxxx&action=edit`

Asking them to update to the 5.6 to see if something changes :)

**Comment by Umangvaghela** [comment]
@StefMattana 

Hello Mam,

Issue is still happen by 
```
function is_post_type_allowed( $post_id ) {
		$post = get_post( $post_id );

		return ! in_array( $post->post_type, Jetpack_Sync_Settings::get_setting( 'post_types_blacklist' ) );
}
```

@jeherve  

Hello Sir

If we update the code so we will overcome the problem.
My solution.

```
function is_post_type_allowed( $post_id ) {
		$post = get_post(intval( $post_id) );
                if( $post->post_type ) { 
                  		return ! in_array( $post->post_type, Jetpack_Sync_Settings::get_setting( 'post_types_blacklist' ); 
                } 
                return false;
}

```

Thanks

**Comment by StefMattana** [comment]
User on [https://wordpress.org/support/topic/trying-to-get-property-of-non-object-49/#post-9787834](https://wordpress.org/support/topic/trying-to-get-property-of-non-object-49/#post-9787834 )fixed it by updating to 5.6.1

**Comment by oskosk** [comment]
Looks like the original issue was fixed by #8143. The issue reported in https://github.com/Automattic/jetpack/issues/8137#issuecomment-351979760 was fixed by #8367
